### ISSUE TITLE
[Documentation Issue] 

### ISSUE THREAD

**Issue by raybuhr** [GoogleCloudPlatform/gcloud-python#1203]
Page Name: bigquery-usage
Release: 0.8.0

Looking at the section for inserting data asynchronously, the load_from_storage method doesn't exist in the table class. There is a load_table_from_storage method in client class but it doesn't have the same args. Is this feature not yet released or just improperly documented?

Inserting data (asynchronous)
Start a job loading data asynchronously from a set of CSV files, located on Google Cloud Storage, appending rows into an existing table. First, create the job locally:

```python
>>> from gcloud import bigquery
>>> client = bigquery.Client()
>>> table = dataset.table(name='person_ages')
>>> job = table.load_from_storage(bucket_name='bucket-name',
...                               object_name_glob='object-prefix*',
...                               source_format='CSV',
...                               skip_leading_rows=1,
...                               write_disposition='truncate')
>>> job.job_id
'e3344fba-09df-4ae0-8337-fddee34b3840'
>>> job.type
'load'
>>> job.created
None
>>> job.state
None
```

**Comment by tseaver** [comment]
Thanks for the report!  I will check into this issue today.

**Comment by dhermes** [comment]
@tseaver Where do we stand on this?

**Comment by tseaver** [comment]
Fixed in 4201985c7f3fc9520de9da21e62d6cb25fb7d7e2.  See http://googlecloudplatform.github.io/gcloud-python/latest/bigquery-usage.html#inserting-data-asynchronous

**Comment by dhermes** [comment]
That's what I thought. Thanks
